src/GraphicalModelLearning.jl

```diff
@@ -162,24 +163,24 @@ function learn(samples::Array{T,2}, formulation::RISE, method::NLP) where T <: R
     for current_spin = 1:num_spins
         nodal_stat  = [ samples[k, 1 + current_spin] * (i == current_spin ? 1 : samples[k, 1 + i]) for k=1:num_conf , i=1:num_spins]
 
-        m = Model(solver = method.solver)
+        model = Model(method.solver)
 
-        @variable(m, x[1:num_spins])
-        @variable(m, z[1:num_spins])
+        @variable(model, x[1:num_spins])
+        @variable(model, z[1:num_spins])
 
-        @NLobjective(m, Min,
+        @NLobjective(model, Min,
             sum((samples[k,1]/num_samples)*exp(-sum(x[i]*nodal_stat[k,i] for i=1:num_spins)) for k=1:num_conf) +
             lambda*sum(z[j] for j=1:num_spins if current_spin!=j)
         )
 
         for j in 1:num_spins
-            @constraint(m, z[j] >=  x[j]) #z_plus
-            @constraint(m, z[j] >= -x[j]) #z_minus
+            @constraint(model, z[j] >=  x[j]) #z_plus
+            @constraint(model, z[j] >= -x[j]) #z_minus
         end
 
-        status = solve(m)
-        @assert status == :Optimal
-        reconstruction[current_spin,1:num_spins] = deepcopy(getvalue(x))
+        JuMP.optimize!(model)
+        @assert JuMP.termination_status(model) == MOI.LOCALLY_SOLVED
```

Comment:
Because this is for internal correctness checking how about `JuMP.MOI. LOCALLY_SOLVED` then we can drop the dep on MOI in this package.